parent/pom.xml

```diff
@@ -105,7 +105,7 @@
     <spotbugs.version>4.0.0-beta3</spotbugs.version>
     <spotbugs.maven.version>3.1.12</spotbugs.maven.version>
     <proton-j-version>0.31.0</proton-j-version>
-    <qpid-proton-j-extensions-version>1.2.0</qpid-proton-j-extensions-version>
+    <qpid-proton-j-extensions-version>1.2.1</qpid-proton-j-extensions-version>
```

Comment:
Since we making changes to the versions here, we should increment the parent version too and update the modules that would need these new versions.